web-console/future-openshift/toolbars/toolbars.md

```diff
@@ -0,0 +1,48 @@
+# Toolbars
+
+There are various toolbars used across the OpenShift console.
+
+
+## Basic toolbars
+* The most basic toolbars in OpenShift have a 'Create' button, a textbox filter, and an item count.
+
+Example
+![deployments](img/deployments-today.png)
+
+## Toolbars with faceted filters
+* This type of toolbar is similar to the basic toolbar but adds faceted filters. When filters are applied, the toolbar increases in height to show filter chips on the second row.
+* Toolbars with faceted filters may or may not have a 'Create' button.
+* Toolbars with faceted filters may have opinionated views. In these cases, filters are already applied when a user navigates to that page, and the filter chips are visible to show what is being filtered.
+* Faceted filters may allow users to filter on more than one attribute where necessary.
+
+Example with 'Create' button and opinionated view
+![pods](img/pods-today.png)
+
+Example without 'Create' button and no opinionated view
+![nodes](img/nodes-today.png)
+
+Faceted filter with multiple attributes
+![nodes2](img/nodes-future.png)
+* Faceted filters can also be used to filter on multiple attributes in a list.
+* Each attribute will have a section with its appropriate values.
+
+## Special toolbars
+* There are a number of pages that have custom toolbars.
+
+Search
+![search](img/search-today.png)
+
+Events
+![events](img/events-today.png)
```

Comment:
Just randomly spot this PR from my inbox. A couple of things worth mentioning:

1. If the project bar is selected to "_default_" as shown in the mock, users can only view Events from "_default_" namespace, not from mutiple namespaces.

2. If the mockup is intended to showcase viewing Events from "all projects", can you take out "_Tony_" from the namespace column? 

Thanks

Reply:
Good catch, will update. Thanks Tony!